README.md

```diff
@@ -32,16 +32,16 @@ See the output of `deepspeech -h` for more information on the use of `deepspeech
 
 - [Prerequisites](#prerequisites)
 - [Getting the code](#getting-the-code)
-- [Getting the pre-trained model](#getting-the-pre-trained-model)
-- [CUDA dependency](#cuda-dependency)
-- [Using the model](#using-the-model)
+- [Using a Pre-trained Model](#using-a-pre-trained-model)
+  - [CUDA dependency](#cuda-dependency)
+  - [Getting the pre-trained model](#downloading-the-pre-trained-model)
   - [Using the Python package](#using-the-python-package)
-  - [Using the command line client](#using-the-command-line-client)
   - [Using the Node.JS package](#using-the-nodejs-package)
+  - [Using the Command Line client](#using-the-command-line-client)
   - [Installing bindings from source](#installing-bindings-from-source)
   - [Third party bindings](#third-party-bindings)
-- [Training](#training)
-  - [Installing prerequisites for training](#installing-prerequisites-for-training)
+- [Training your own Model](#training)
+  - [Installing training prerequisites](#installing-prerequisites-for-training)
```

Comment:
The section titles should be updated to match the TOC.

Reply:
fixed: 7ab0962b3d31c0d9f89238da2962c2edce998e7b